---
layout: site
title: BEAST 2 Help Me Choose
tags: []
---

# Welcome to Help Me Choose

## Standard template
<br><a href='/hmc/hmc/Standard/Clock_Model/'>Standard template -- clock model panel</a>
<br><a href='/hmc/hmc/Standard/Initialization/'>Standard template -- initialization panel</a>
<br><a href='/hmc/hmc/Standard/MCMC/'>Standard template -- MCMC panel</a>
<br><a href='/hmc/hmc/Standard/Operators/'>Standard template -- operators panel</a>
<br><a href='/hmc/hmc/Standard/Partitions/'>Standard template -- partitions panel</a>
<br><a href='/hmc/hmc/Standard/Priors/'>Standard template -- priors panel</a>
<br><a href='/hmc/hmc/Standard/Site_Model/'>Standard template -- site model panel</a>
<br><a href='/hmc/hmc/Standard/Starting_tree/'>Standard template -- starting tree panel</a>
<br><a href='/hmc/hmc/Standard/Tip_Dates/'>Standard template -- tip dates panel</a>

## Priors
<br><a href='/hmc/hmc/Priors/BirthRatePrior/'>Birth rate prior</a>
<br><a href='/hmc/hmc/Priors/CalibratedYuleBirthRatePrior/'>Calibrated Yule birth rate prior</a>
<br><a href='/hmc/hmc/Priors/ClockPrior/'>Clock prior</a>
<br><a href='/hmc/hmc/Priors/DeathRatePrior/'>Death rate prior</a>
<br><a href='/hmc/hmc/Priors/ePopSizePrior/'>Exponential pop size prior</a>
<br><a href='/hmc/hmc/Priors/FrequenciesPrior/'>Frequencies prior</a>
<br><a href='/hmc/hmc/Priors/GammaShapePrior/'>Gamma shape prior</a>
<br><a href='/hmc/hmc/Priors/GrowthRatePrior/'>Growth rate prior</a>
<br><a href='/hmc/hmc/Priors/indicatorsPrior/'>Indicators prior</a>
<br><a href='/hmc/hmc/Priors/kappa1Prior/'>Kappa 1 prior</a>
<br><a href='/hmc/hmc/Priors/kappa2Prior/'>Kappa 2 prior</a>
<br><a href='/hmc/hmc/Priors/KappaPrior/'>Kappa prior</a>
<br><a href='/hmc/hmc/Priors/MRCAPrior/'>MRCA prior</a>
<br><a href='/hmc/hmc/Priors/MutationRatePrior/'>Mutation rate prior</a>
<br><a href='/hmc/hmc/Priors/PopMean/'>Pop mean prior</a>
<br><a href='/hmc/hmc/Priors/PopSizePrior/'>Pop size prior</a>
<br><a href='/hmc/hmc/Priors/populationMeanPrior/'>Population mean prior</a>
<br><a href='/hmc/hmc/Priors/PropInvariantPrior/'>Proportion invariable prior</a>
<br><a href='/hmc/hmc/Priors/RateGTRPrior/'>GTR rate priors</a>
<br><a href='/hmc/hmc/Priors/YuleBirthRatePrior/'>Yule birth rate prior</a>
<br>See <a href='/hmc/hmc/Standard/Priors/'>here</a> for tree priors.

## Parametric Distributions
<br><a href='/hmc/hmc/ParametricDistributions/Beta/alpha/'>Beta alpha</a>
<br><a href='/hmc/hmc/ParametricDistributions/Beta/beta/'>Beta beta</a>
<br><a href='/hmc/hmc/ParametricDistributions/Beta/offset/'>Beta offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/Dirichlet/alpha/'>Dirichlet alpha</a>
<br><a href='/hmc/hmc/ParametricDistributions/Dirichlet/offset/'>Dirichlet offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/Exponential/mean/'>Exponential mean</a>
<br><a href='/hmc/hmc/ParametricDistributions/Exponential/offset/'>Exponential offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/Gamma/alpha/'>Gamma alpha</a>
<br><a href='/hmc/hmc/ParametricDistributions/Gamma/beta/'>Gamma beta</a>
<br><a href='/hmc/hmc/ParametricDistributions/Gamma/mode/'>Gamma mode</a>
<br><a href='/hmc/hmc/ParametricDistributions/Gamma/offset/'>Gamma offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/InverseGamma/alpha/'>Inverse Gamma alpha</a>
<br><a href='/hmc/hmc/ParametricDistributions/InverseGamma/beta/'>Inverse Gamma beta</a>
<br><a href='/hmc/hmc/ParametricDistributions/InverseGamma/offset/'>Inverse Gamma offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/LaplaceDistribution/mu/'>Laplace Distribution mu</a>
<br><a href='/hmc/hmc/ParametricDistributions/LaplaceDistribution/offset/'>Laplace Distribution offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/LaplaceDistribution/scale/'>Laplace Distribution scale</a>
<br><a href='/hmc/hmc/ParametricDistributions/LogNormalDistributionModel/M/'>Log Normal Distribution Model  M</a>
<br><a href='/hmc/hmc/ParametricDistributions/LogNormalDistributionModel/meanInRealSpace/'>Log Normal Distribution Model mean In Real Space</a>
<br><a href='/hmc/hmc/ParametricDistributions/LogNormalDistributionModel/offset/'>Log Normal Distribution Model offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/LogNormalDistributionModel/S/'>Log Normal Distribution Model  S</a>
<br><a href='/hmc/hmc/ParametricDistributions/Normal/mean/'>Normal mean</a>
<br><a href='/hmc/hmc/ParametricDistributions/Normal/offset/'>Normal offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/Normal/sigma/'>Normal sigma</a>
<br><a href='/hmc/hmc/ParametricDistributions/OneOnX/offset/'>One On X offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/Poisson/lambda/'>Poisson lambda</a>
<br><a href='/hmc/hmc/ParametricDistributions/Poisson/offset/'>Poisson offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/Uniform/lower/'>Uniform lower</a>
<br><a href='/hmc/hmc/ParametricDistributions/Uniform/offset/'>Uniform offset</a>
<br><a href='/hmc/hmc/ParametricDistributions/Uniform/upper/'>Uniform upper</a>

## Operators
<br><a href='/hmc/hmc/Operators/BactrianDeltaExchange/'>Bactrian delta exchange operator</a>
<br><a href='/hmc/hmc/Operators/BactrianDistribution/'>Kernel distribution</a>
<br><a href='/hmc/hmc/Operators/BactrianNodeOperator/'>Bactrian node operator</a>
<br><a href='/hmc/hmc/Operators/BactrianRandomWalk/'>Bactrian random walk operator</a>
<br><a href='/hmc/hmc/Operators/BactrianRateScaler/'>Bactrian rate scale operator</a>
<br><a href='/hmc/hmc/Operators/BactrianSubtreeSlide/'>Subtree slide operator</a>
<br><a href='/hmc/hmc/Operators/BactrianTreeScaler/'>Bactrian tree scale operator</a>
<br><a href='/hmc/hmc/Operators/BactrianUpDown/'>Bactrian up down operator</a>
<br><a href='/hmc/hmc/Operators/BitFlipper/'>Bit flip operator</a>
<br><a href='/hmc/hmc/Operators/EpochFlexer/'>Epoch flex operator</a>
<br><a href='/hmc/hmc/Operators/Exchange/'>Exchange operator</a>
<br><a href='/hmc/hmc/Operators/OperatoreWeights/'>Operator weight tuning</a>
<br><a href='/hmc/hmc/Operators/SampleOffValues/'>SampleOffValues operator</a>
<br><a href='/hmc/hmc/Operators/SubtreeSlide/'>Subtree slide operator</a>
<br><a href='/hmc/hmc/Operators/TreeStretcher/'>Tree stretch operator</a>
<br><a href='/hmc/hmc/Operators/WilsonBalding/'>Wilson Balding operator</a>

## Other
<br><a href='/hmc/hmc/BayesianSkyline/index/'>Bayesian skyline</a>
<br><a href='/hmc/hmc/BirthDeath/index/'>Birth death</a>
<br><a href='/hmc/hmc/birthRate/dimension/'>Birth rate -- dimension</a>
<br><a href='/hmc/hmc/birthRate/estimate/'>Birth rate -- estimate</a>
<br><a href='/hmc/hmc/birthRate/keys/'>Birth rate -- keys</a>
<br><a href='/hmc/hmc/birthRate/lower/'>Birth rate -- lower</a>
<br><a href='/hmc/hmc/birthRate/minordimension/'>Birth rate -- minordimension</a>
<br><a href='/hmc/hmc/birthRate/upper/'>Birth rate -- upper</a>
<br><a href='/hmc/hmc/CalibratedYuleModel/index/'>Calibrated Yule model</a>
<br><a href='/hmc/hmc/ClusterTree/adjustTreeNodeHeights/'>Cluster tree -- adjust tree node heights</a>
<br><a href='/hmc/hmc/ClusterTree/clock.rate/'>Cluster tree -- clock.rate</a>
<br><a href='/hmc/hmc/ClusterTree/clusterType/'>Cluster tree -- cluster type</a>
<br><a href='/hmc/hmc/CoalescentConstant/index/'>Coalescent constant</a>
<br><a href='/hmc/hmc/CoalescentExponential/index/'>Coalescent exponential</a>
<br><a href='/hmc/hmc/ConstantPopulation0/popSize/'>Constant population0 pop size</a>
<br><a href='/hmc/hmc/EBSPLogger/fileName/'>EBSP logger -- file name</a>
<br><a href='/hmc/hmc/EBSPLogger/logEvery/'>EBSP logger -- log every</a>
<br><a href='/hmc/hmc/EBSPLogger/mode/'>EBSP logger -- mode</a>
<br><a href='/hmc/hmc/EBSPLogger/sanitiseHeaders/'>EBSP logger -- sanitise headers</a>
<br><a href='/hmc/hmc/EBSPLogger/sort/'>EBSP logger -- sort</a>
<br><a href='/hmc/hmc/ExtendedBayesianSkyline/index/'>Extended Bayesian skyline</a>
<br><a href='/hmc/hmc/gtr/frequencies/'>GTR frequencies</a>
<br><a href='/hmc/hmc/gtr/rateAC/'>GTR rate AC</a>
<br><a href='/hmc/hmc/gtr/rateAG/'>GTR rate AG</a>
<br><a href='/hmc/hmc/gtr/rateAT/'>GTR rate AT</a>
<br><a href='/hmc/hmc/gtr/rateCG/'>GTR rate CG</a>
<br><a href='/hmc/hmc/gtr/rateCT/'>GTR rate CT</a>
<br><a href='/hmc/hmc/gtr/rateGT/'>GTR rate GT</a>
<br><a href='/hmc/hmc/hky/frequencies/'>HKY frequencies</a>
<br><a href='/hmc/hmc/hky/kappa/'>HKY kappa</a>
<br><a href='/hmc/hmc/MarkovChainedPopSizes/initialMean/'>Markov chained pop sizes -- initial mean</a>
<br><a href='/hmc/hmc/MarkovChainedPopSizes/jeffreys/'>Markov chained pop sizes -- jeffreys</a>
<br><a href='/hmc/hmc/MarkovChainedPopSizes/parameter/'>Markov chained pop sizes -- parameter</a>
<br><a href='/hmc/hmc/MarkovChainedPopSizes/reverse/'>Markov chained pop sizes -- reverse</a>
<br><a href='/hmc/hmc/MarkovChainedPopSizes/shape/'>Markov chained pop sizes -- shape</a>
<br><a href='/hmc/hmc/MarkovChainedPopSizes/uselog/'>Markov chained pop sizes -- uselog</a>
<br><a href='/hmc/hmc/MarkovChainedPopSizes/useLogNormal/'>Markov chained pop sizes -- use Log Normal</a>
<br><a href='/hmc/hmc/mcmc/chainLength/'>MCMC chain length</a>
<br><a href='/hmc/hmc/mcmc/numInitializationAttempts/'>MCMC num initialization attempts</a>
<br><a href='/hmc/hmc/mcmc/operatorschedule/'>MCMC operator schedule</a>
<br><a href='/hmc/hmc/mcmc/preBurnin/'>MCMC pre burnin</a>
<br><a href='/hmc/hmc/mcmc/sampleFromPrior/'>MCMC sample from prior</a>
<br><a href='/hmc/hmc/mcmc/storeEvery/'>MCMC store every</a>
<br><a href='/hmc/hmc/NewickTree/adjustTipHeights/'>Newick tree -- adjust tip heights</a>
<br><a href='/hmc/hmc/NewickTree/adjustTreeNodeHeights/'>Newick tree -- adjust tree node heights</a>
<br><a href='/hmc/hmc/NewickTree/binarizeMultifurcations/'>Newick tree -- binarize multifurcations</a>
<br><a href='/hmc/hmc/NewickTree/IsLabelledNewick/'>Newick tree -- is labelled Newick</a>
<br><a href='/hmc/hmc/NewickTree/newick/'>Newick tree -- newick</a>
<br><a href='/hmc/hmc/NewickTree/offset/'>Newick tree -- offset</a>
<br><a href='/hmc/hmc/NewickTree/scale/'>Newick tree -- scale</a>
<br><a href='/hmc/hmc/NewickTree/singlechild/'>Newick tree - singlechild</a>
<br><a href='/hmc/hmc/NewickTree/threshold/'>Newick tree -- threshold</a>
<br><a href='/hmc/hmc/OperatorSchedule/autoOptimize/'>Operator schedule -- auto optimize</a>
<br><a href='/hmc/hmc/OperatorSchedule/autoOptimizeDelay/'>Operator schedule -- auto optimize delay</a>
<br><a href='/hmc/hmc/OperatorSchedule/detailedRejection/'>Operator schedule -- detailed rejection</a>
<br><a href='/hmc/hmc/OperatorSchedule/transform/'>Operator schedule --- transform</a>
<br><a href='/hmc/hmc/OperatorSchedule/weight/'>Operator schedule -- weight</a>
<br><a href='/hmc/hmc/OperatorSchedule/weightIsPercentage/'>Operator schedule -- weight is percentage</a>
<br><a href='/hmc/hmc/RandomLocalClock/includeRoot/'>Random local clock -- include root</a>
<br><a href='/hmc/hmc/RandomLocalClock/ratesAreMultipliers/'>Random local clock -- rates are multipliers</a>
<br><a href='/hmc/hmc/RandomLocalClock/RatesPrior/'>Random local clock -- rate prior</a>
<br><a href='/hmc/hmc/RandomLocalClock/RRateChangesPrior/'>Random local clock -- rate change prior</a>
<br><a href='/hmc/hmc/RandomLocalClock/scaling/'>Random local clock -- scaling</a>
<br><a href='/hmc/hmc/RandomTree/adjustTreeNodeHeights/'>Random tree -- adjust tree node heights</a>
<br><a href='/hmc/hmc/RandomTree/rootHeight/'>Random tree -- root height</a>
<br><a href='/hmc/hmc/RelaxedClock/normalize/'>Relaxed clock -- normalize</a>
<br><a href='/hmc/hmc/RelaxedClock/numberOfDiscreteRates/'>Relaxed Clock -- number of discrete rates</a>
<br><a href='/hmc/hmc/screenlog/fileName/'>Screen log -- file Name</a>
<br><a href='/hmc/hmc/screenlog/logEvery/'>Screen log -- log every</a>
<br><a href='/hmc/hmc/screenlog/mode/'>Screen log -- mode</a>
<br><a href='/hmc/hmc/screenlog/sanitiseHeaders/'>Screen log -- sanitise headers</a>
<br><a href='/hmc/hmc/screenlog/sort/'>Screen log -- sort</a>
<br><a href='/hmc/hmc/SiteModel/gammaCategoryCount/'>Site Model gamma category count</a>
<br><a href='/hmc/hmc/SiteModel/mutationRate/'>Site Model mutation rate</a>
<br><a href='/hmc/hmc/SiteModel/proportionInvariant/'>Site Model proportion invariable</a>
<br><a href='/hmc/hmc/SiteModel/shape/'>Site Model shape</a>
<br><a href='/hmc/hmc/SiteModel/substModel/'>Site model subst model</a>
<br><a href='/hmc/hmc/speciesTreeLogger/fileName/'>Species tree logger -- file name</a>
<br><a href='/hmc/hmc/speciesTreeLogger/logEvery/'>Species tree logger -- log every</a>
<br><a href='/hmc/hmc/speciesTreeLogger/mode/'>Species tree logger -- mode</a>
<br><a href='/hmc/hmc/speciesTreeLogger/sanitiseHeaders/'>Species tree logger -- sanitise headers</a>
<br><a href='/hmc/hmc/speciesTreeLogger/sort/'>Species tree logger -- sort</a>
<br><a href='/hmc/hmc/StrictClock/clock.rate/'>clock.rate</a>
<br><a href='/hmc/hmc/tn93/frequencies/'>TN93 frequencies</a>
<br><a href='/hmc/hmc/tn93/kappa1/'>tn93 kappa1</a>
<br><a href='/hmc/hmc/tn93/kappa2/'>tn93 kappa2</a>
<br><a href='/hmc/hmc/tracelog/fileName/'>Trace log -- file name</a>
<br><a href='/hmc/hmc/tracelog/logEvery/'>Trace log -- log every</a>
<br><a href='/hmc/hmc/tracelog/mode/'>Trace log -- mode</a>
<br><a href='/hmc/hmc/tracelog/sanitiseHeaders/'>Trace log -- sanitise headers</a>
<br><a href='/hmc/hmc/tracelog/sort/'>Trace log -- sort</a>
<br><a href='/hmc/hmc/treeLikelihood/branchRateModel/'>Tree likelihood branch rate model</a>
<br><a href='/hmc/hmc/treeLikelihood/siteModel/'>Tree likelihood site model</a>
<br><a href='/hmc/hmc/treelog/fileName/'>Tree log -- file name</a>
<br><a href='/hmc/hmc/treelog/logEvery/'>Tree log -- log every</a>
<br><a href='/hmc/hmc/treelog/mode/'>Tree log -- mode</a>
<br><a href='/hmc/hmc/treelog/sanitiseHeaders/'>Tree log -- sanitise headers</a>
<br><a href='/hmc/hmc/treelog/sort/'>Tree log -- sort</a>
<br><a href='/hmc/hmc/YuleModel/index/'>Yule model</a>

## References
<br>Banner KM, Irvine KM, Rodhouse TJ. The use of Bayesian priors in Ecology: The good, the bad and the not great. Methods in Ecology and Evolution. 2020 Aug;11(8):882-9. <a href="https://doi.org/10.1111/2041-210X.13407">doi:10.1111/2041-210X.13407</a>.

<br>Bouckaert RR. An Efficient Coalescent Epoch Model for Bayesian Phylogenetic Inference, Systematic Biology, syac015, 2022 <a href="https://doi.org/10.1093/sysbio/syac015">doi:10.1093/sysbio/syac015</a>.

<br>Bouckaert RR. OBAMA: OBAMA for Bayesian amino-acid model averaging. PeerJ. 2020 Aug 4;8:e9460. <a href="http://doi.org/10.7717/peerj.9460">doi:10.7717/peerj.9460</a>.

<br>Bouckaert RR, Drummond AJ. bModelTest: Bayesian phylogenetic site model averaging and model comparison. BMC evolutionary biology. 2017 Dec;17(1):1-1. <a href="http://doi.org/10.1186/s12862-017-0890-6">doi:10.1186/s12862-017-0890-6</a>.

<br>Bouckaert R, Heled J, Kühnert D, Vaughan T, Wu CH, Xie D, Suchard MA, Rambaut A, Drummond AJ. BEAST 2: a software platform for Bayesian evolutionary analysis. PLoS computational biology. 2014 Apr 10;10(4):e1003537. <a href="http://doi.org/10.1371/journal.pcbi.1003537">doi:10.1371/journal.pcbi.1003537</a>

<br>Bouckaert R, Vaughan TG, Barido-Sottani J, Duchêne S, Fourment M, Gavryushkina A, Heled J, Jones G, Kühnert D, De Maio N, Matschiner M. BEAST 2.5: An advanced software platform for Bayesian evolutionary analysis. PLoS computational biology. 2019 Apr 8;15(4):e1006650. <a href="http://doi.org/10.1371/journal.pcbi.1006650">doi:10.1371/journal.pcbi.1006650</a>

<br>Douglas J, Jiménez-Silva CL, Bouckaert R. StarBeast3: Adaptive Parallelized Bayesian Inference under the Multispecies Coalescent. Systematic Biology. 2022 Feb 17. <a href="http://doi.org/10.1093/sysbio/syac010">doi:10.1093/sysbio/syac010</a>.

<br>Douglas J, Zhang R, Bouckaert R. Adaptive dating and fast proposals: Revisiting the phylogenetic relaxed clock model. PLoS computational biology. 2021 Feb 2;17(2):e1008322. <a href="https://doi.org/10.1371/journal.pcbi.1008322">doi:10.1371/journal.pcbi.1008322</a>.

<br>Drummond AJ, Ho SY, Phillips MJ, Rambaut A. Relaxed phylogenetics and dating with confidence. PLoS biology. 2006 May;4(5):e88. <a href="https://doi.org/10.1371/journal.pbio.0040088">doi:10.1371/journal.pbio.0040088</a>

<br>Drummond AJ, Rambaut A, Shapiro BE, Pybus OG. Bayesian coalescent inference of past population dynamics from molecular sequences, Molecular Biology and Evolution. 22(5):1185-92, 2005. <a href="http://doi.org/10.1093/molbev/msi103">doi:10.1093/molbev/msi103</a>.

<br>Drummond AJ, Suchard MA. Bayesian random local clocks, or one rate to rule them all. BMC biology. 2010 Dec;8(1):1-2. <a href="https://doi.org/10.1186/s12862-020-01609-4">doi:10.1186/s12862-020-01609-4</a>.

<br>Fagundes De Carvalho LM. Statistical approaches to viral phylodynamics (Doctoral dissertation, University of Edinburgh). 2018. <a href="http://hdl.handle.net/1842/35510">online</a>.

<br>Frankham R. Effective population size/adult population size ratios in wildlife: a review. Genetics Research. 1995 Oct;66(2):95-107. <a href="https://doi.org/10.1017/S0016672300034455">doi:10.1017/S0016672300034455</a>.

<br>Gavryushkina A, Welch D, Stadler T, Drummond AJ. Bayesian inference of sampled ancestor trees for epidemiology and fossil calibration. PLoS computational biology. 2014 Dec 4;10(12):e1003919. <a href="https://doi.org/10.1371/journal.pcbi.1003919">doi:10.1371/journal.pcbi.1003919</a>.

<br>Gernhard, T. 2008. The conditioned reconstructed process. Journal of Theoretical Biology Volume 253, Issue 4, 21 August 2008, Pages 769-778 <a href="http://doi.org/10.1016/j.jtbi.2008.04.005">doi:10.1016/j.jtbi.2008.04.005</a>.

<br>Hasegawa M, Kishino H, Yano TA. Dating of the human-ape splitting by a molecular clock of mitochondrial DNA. Journal of molecular evolution. 1985 Oct;22(2):160-74. <a href="https://doi.org/10.1007%2FBF02101694"> doi:10.1007/BF02101694</a>.

<br>Heled J, Drummond AJ. Calibrated tree priors for relaxed phylogenetics and divergence time estimation. Systematic biology. 2012 Jan 1;61(1):138-49. <a href="https://doi.org/10.1093/sysbio/syr087">doi:10.1093/sysbio/syr087</a>.

<br>Heled J, Drummond AJ. Bayesian inference of population size history from multiple loci. BMC evolutionary biology. 2008 Dec;8(1):1-5. <a href="https://doi.org/10.1186/1471-2148-8-289">doi:10.1186/1471-2148-8-289</a>.

<br>Jenkins GM, Rambaut A, Pybus OG, Holmes EC. Rates of molecular evolution in RNA viruses: a quantitative phylogenetic analysis. Journal of molecular evolution. 2002 Feb;54(2):156-65. <a href="http://doi.org/10.1007/s00239-001-0064-3">doi:10.1007/s00239-001-0064-3</a>

<br>Kühnert D, Stadler T, Vaughan TG, Drummond AJ. Phylodynamics with migration: a computational framework to quantify population structure from genomic data. Molecular biology and evolution. 2016 Apr 9;33(8):2102-16. <a href="https://doi.org/10.1093/molbev/msw064">doi:10.1093/molbev/msw064</a>.

<br>Krasovec M, Rickaby RE, Filatov DA. Evolution of mutation rate in astronomically large phytoplankton populations. Genome biology and evolution. 2020 Jul 1. <a href="http://dx.doi.org/10.1093/gbe/evaa131">doi:10.1093/gbe/evaa131</a>.

<br>Kumar S, Subramanian S. Mutation rates in mammalian genomes. Proceedings of the National Academy of Sciences. 2002 Jan 22;99(2):803-8. <a href="http://doi.org/10.1073/pnas.022629899">doi:10.1073/pnas.022629899</a>.

<br>Lunn D, Jackson C, Best N, Thomas A, Spiegelhalter D. The BUGS book. A Practical Introduction to Bayesian Analysis, Chapman Hall, London. 2013.

<br>Magallon S, Sanderson MJ. Absolute diversification rates in angiosperm clades. Evolution. 2001 Sep;55(9):1762-80. <a href="http://doi.org/10.1111/j.0014-3820.2001.tb00826.x">doi:10.1111/j.0014-3820.2001.tb00826.x</a>.

<br>Matschiner M, Musilová Z, Barth JM, Starostová Z, Salzburger W, Steel M, Bouckaert R. Bayesian phylogenetic estimation of clade ages supports trans-Atlantic dispersal of cichlid fishes. Systematic biology. 2017 Jan 1;66(1):3-22. <a href="https://doi.org/10.1093/sysbio/syw076">doi:10.1093/sysbio/syw076</a>.

<br>Maturana PM, Brewer BJ, Klaere S, Bouckaert RR. Model selection and parameter inference in phylogenetics using Nested Sampling. Systematic biology. 2019 Mar 1;68(2):219-33. <a href="https://doi.org/10.1093/sysbio/syy050">doi:10.1093/sysbio/syy050</a>.

<br>Müller NF, Bouckaert RR. Adaptive Metropolis-coupled MCMC for BEAST 2. PeerJ. 2020 Sep 16;8:e9473. <a href="https://doi.org/10.7717/peerj.9473">doi:10.7717/peerj.9473</a>.

<br>Müller NF, Rasmussen D, Stadler T. MASCOT: parameter and state inference under the marginal structured coalescent approximation. Bioinformatics. 2018 Nov 15;34(22):3843-8. <a href="http://doi.org/10.1093/bioinformatics/bty406">doi:10.1093/bioinformatics/bty406</a>.

<br>Ogilvie HA, Heled J, Xie D, Drummond AJ. Computational performance and statistical accuracy of *BEAST and comparisons with other methods. Systematic biology. 2016 May 1;65(3):381-96. <a href="http://doi.org/10.1093/sysbio/syv118">doi:10.1093/sysbio/syv118</a>.

<br>Rosenberg MS, Subramanian S, Kumar S. Patterns of transitional mutation biases within and among mammalian genomes. Molecular biology and evolution. 2003 Jun 1;20(6):988-93. <a href="https://doi.org/10.1093/molbev/msg113">doi:10.1093/molbev/msg113</a>.

<br>Sanjuán R, Nebot MR, Chirico N, Mansky LM, Belshaw R. Viral mutation rates. Journal of virology. 2010 Oct 1;84(19):9733-48. <a href="http://doi.org/10.1128/JVI.00694-10">doi:10.1128/JVI.00694-10</a>.

<br>Sarma A, Kay M. Prior setting in practice: Strategies and rationales used in choosing prior distributions for Bayesian analysis. Proceedings of the 2020 chi conference on human factors in computing systems 2020 Apr 21 (pp. 1-12). <a href="http://doi.org/10.1145/3313831.3376377">doi: 10.1145/3313831.3376377</a>.

<br>Shapiro B, Rambaut A, Drummond AJ. Choosing appropriate substitution models for the phylogenetic analysis of protein-coding sequences. Molecular biology and evolution. 2006 Jan 1;23(1):7-9. <a href="http://doi.org/10.1093/molbev/msj021">doi:10.1093/molbev/msj021</a>.

<br>Stadler T, Kühnert D, Bonhoeffer S, Drummond AJ. Birth–death skyline plot reveals temporal changes of epidemic spread in HIV and hepatitis C virus (HCV). Proceedings of the National Academy of Sciences. 2013 Jan 2;110(1):228-33. <a href="https://doi.org/10.1073/pnas.120796511">doi:10.1073/pnas.120796511</a>.

<br>Steel M, Mooers A. The expected length of pendant and interior edges of a Yule tree. Applied Mathematics Letters. 2010 Nov 1;23(11):1315-9. <a href="https://doi.org/10.1016/j.aml.2010.06.021">doi:10.1016/j.aml.2010.06.021</a>.

<br>Stoltz M, Baeumer B, Bouckaert R, Fox C, Hiscott G, Bryant D. Bayesian inference of species trees using diffusion models. Systematic Biology. 2021 Jan;70(1):145-61. <a href="http://doi.org/10.1093/sysbio/syaa051">doi:10.1093/sysbio/syaa051</a>.

<br>Tamura K, Nei M. Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees. Molecular biology and evolution. 1993 May 1;10(3):512-26. <a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a040023">doi:10.1093/oxfordjournals.molbev.a040023</a>.

<br>Tavaré S, Balding DJ, Griffiths RC, Donnelly P. Inferring coalescence times from DNA sequence data. Genetics. 1997 Feb 1;145(2):505-18. <a href="http://doi.org/10.1093/genetics/145.2.505">doi:10.1093/genetics/145.2.505</a>.

<br>Tavaré S. Some probabilistic and statistical problems in the analysis of DNA sequences. Lectures on mathematics in the life sciences. 1986 Dec 31;17(2):57-86.

<br>Waddell P. Evolutionary trees of apes and humans from DNA sequences. Handbook of human symbolic evolution. 1996:53-73.

<br>Vaughan TG, Kühnert D, Popinga A, Welch D, Drummond AJ. Efficient Bayesian inference under the structured coalescent. Bioinformatics. 2014 Aug 15;30(16):2272-9. <a href="https://doi.org/10.1093/bioinformatics/btu201">doi:10.1093/bioinformatics/btu201</a>.

<br>Yang J, Müller NF, Bouckaert R, Xu B, Drummond AJ. Bayesian phylodynamics of avian influenza A virus H9N2 in Asia with time-dependent predictors of migration. PLoS computational biology. 2019 Aug 6;15(8):e1007189. <a href="https://doi.org/10.1371/journal.pcbi.1007189">doi:10.1371/journal.pcbi.1007189</a>.

<br>Yang Z. Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: approximate methods. Journal of Molecular evolution. 1994 Sep;39(3):306-14. <a href="http://doi.org/10.1007/BF00160154">doi:10.1007/BF00160154</a>.

<br>Zhang R, Drummond A. Improving the performance of Bayesian phylogenetic inference under relaxed clock models. BMC evolutionary biology. 2020 Dec;20(1):1-28. <a href="http://doi.org/10.1186/s12862-020-01609-4">doi:10.1186/s12862-020-01609-4</a>.

<br>

<br>

